### DISCUSSION TITLE
Why use getCoreRowModel?

### DISCUSSION THREAD

**Discussion by ivolkoff** [TanStack/table#4993] [General]
In all examples, you need to include `getCoreRowModel`. 

This is a standard connection that is always used, why can't it be made by default during initialization?

**Comment by 9mm** [comment]
Yes I also wonder this, theres no good explanation i can find what the point of this is

**Comment by zibiv** [comment]
> In all examples, you need to include `getCoreRowModel`.
> 
> This is a standard connection that is always used, why can't it be made by default during initialization?

I think we use getCoreRowModel explicitly because TanStack Table is framework agnostic . We can use it with React, Vue etc.
Let's say you import your getCoreRowModel fn from @tanstack/react-table. @tanstack/react-table is adapter lib for React. But getCoreRowModel function originally goes from @tanstack/table-core. You can use this default function for getCoreRowModel in useReactTable or use your custom function.
